### ISSUE TITLE
Goes NaN if you reach a certain distance away from the actual city?

### ISSUE THREAD

**Issue by TheBigCon4800** [jpwright/subway#9]
As a joke, i extended the 7 line 69 or so miles away but when the station was placed the stats for ridership, cost of a metrocard etc just went to NaN.

**Comment by jpwright** [comment]
Yup, the ridership estimates only work within a certain range of NYC. I'll fix this to error more gracefully. Thanks!

**Comment by Furness804** [comment]
Will you apply this algorithm to other cities, ie London, Paris, Vancouver, Boston, etc? The ability to add your own lines negates the issue of having to add new sets of lines, so could the algorithm simply be applied to select cities?

**Comment by bootmii** [comment]
It still NaNs in eastern Hicksville.
